sklearn/datasets/lfw.py

```diff
@@ -146,7 +146,7 @@ def _load_imgs(file_paths, slice_, color, resize):
     if not color:
         faces = np.zeros((n_faces, h, w), dtype=np.float32)
     else:
-        faces = np.zeros((n_faces, h, w, 3), dtype=np.float32)
+        faces = np.zeros((n_faces, int(h), int(w), 3), dtype=np.float32)
```

Comment:
I don't understand. Why can these be floats? Also, why only in the color branch?